pipeline.py

```diff
@@ -51,19 +62,26 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path = Path(self.path_to_raw_txt_data)
+
+        for file in path.glob('*'):
+            if '_raw.txt' in file.name:
+                article_id = int(re.search(r'\d+_raw', file.name)[0][:-4])
+                self._storage[article_id] = Article(url=None, article_id=article_id)
```

Comment:
what are those slices about? it is completely not readable and very risk prone: `)[0][:-4])`